src-docs/src/views/guidelines/writing_examples.js

```diff
@@ -0,0 +1,730 @@
+        <GuideRuleExample
+          panelDisplay="block"
+          type="do"
+          text="Place a Learn more after a text that briefly introduces a topic or feature."
+        >
+          <EuiCallOut size="m" title="Building a dashboard?" iconType="gear">
+            <p>
+              Create content directly from our Dashboard app using our new
+              integrated workflow.{' '}
+              <EuiLink href="http://www.elastic.co" external>
+                Learn more.
+              </EuiLink>
+            </p>
+          </EuiCallOut>
+          <EuiSpacer />
+        </GuideRuleExample>
+        <GuideRuleExample
+          panelDisplay="block"
+          type="do"
+          text="Use inline links to point to a specific page that is stable."
+        >
+          <EuiEmptyPrompt
+            title={<h2>Extend your trial</h2>}
+            body={
+              <p>
+                To use advanced security and our other awesome&nbsp;
+                <EuiLink href="http://www.elastic.co">
+                  subscription features
+                </EuiLink>
+                , request an extension now.
+              </p>
+            }
+            actions={
+              <EuiButton color="primary" fill>
+                Extend trial
+              </EuiButton>
+            }
+          />
+        </GuideRuleExample>
+      </GuideRule>
+
+      <EuiHorizontalRule />
+
+      <EuiSpacer size="m" />
+
+      <EuiText>
+        <h2 id="modal">Modals</h2>
+
+        <p>
+          Modals are typically used for confirmation when the user might lose
+          data.
+        </p>
+      </EuiText>
+
+      <GuideRule heading="" description="">
+        <GuideRuleExample
+          type="do"
+          text="For the body text, use one to two short sentences that explain the consequences."
+        >
+          <EuiPanel paddingSize="none" style={modalStyles}>
+            <EuiModalHeader>
+              <EuiModalHeaderTitle>
+                New dashboard already in progress
+              </EuiModalHeaderTitle>
+            </EuiModalHeader>
+            <EuiModalBody>
+              <EuiText>
+                <p>You can continue editing or start with a blank dashboard.</p>
+              </EuiText>
+            </EuiModalBody>
+
+            <EuiModalFooter>
+              <EuiButtonEmpty>Continue editing</EuiButtonEmpty>
+
+              <EuiButton fill>Start over</EuiButton>
+            </EuiModalFooter>
+          </EuiPanel>
+        </GuideRuleExample>
+
+        <GuideRuleExample type="do" text="Some cases requrie three buttons.">
+          <EuiPanel paddingSize="none" style={modalStyles}>
+            <EuiModalHeader>
+              <EuiModalHeaderTitle>
+                You have unsaved changes
+              </EuiModalHeaderTitle>
+            </EuiModalHeader>
+            <EuiModalBody>
+              <EuiText>
+                <p>
+                  You can keep or discard your changes on return to view mode.
```

Comment:
It might just be me, but I don't understand "on return to view mode". For the purposes of a simple example, can we just simplify this statement? Maybe something like: 
```suggestion
                  When you exit editing mode, you can keep or discard your changes.
```